.github/workflows/test.yml

```diff
@@ -39,6 +39,12 @@ jobs:
           - ruby: '3.2'
             activerecord: '7.1'
             gemfile: 'activerecord_7_1.gemfile'
+          - ruby: '3.1'
```

Comment:
Rails 7.2 supports Ruby 3.1 as a minimum version.

https://guides.rubyonrails.org/7_2_release_notes.html

Reply:
I believe that it is enough to test against latest stable version of Ruby